sanic/response/types.py

```diff
@@ -59,7 +59,7 @@ def __repr__(self):
         class_name = self.__class__.__name__
         return f"<{class_name}: {self.status} {self.content_type}>"
 
-    def _encode_body(self, data: Optional[AnyStr]):
+    def _encode_body(self, data: Optional[str | bytes]):
```

Comment:
This was an odd change - without this, everything complained that it should be `Sequence[object]`?

```
sanic/response/types.py:334: error: Value of type variable "AnyStr" of "_encode_body" of "BaseHTTPResponse" cannot be "Sequence[object]"  [type-var]
```

Unfortunately, not familiar enough with mypy for this.